pkg/collector/runner/runner.go

```diff
@@ -149,7 +149,29 @@ func (r *Runner) work() {
 		}
 		r.m.Unlock()
 
-		log.Infof("Running check %s", check)
+		var doLog bool
+		var lastLog bool
+		checkStats.M.RLock()
+		s, found := checkStats.Stats[check.ID()]
+		if found {
```

Comment:
Can we isolate this in a function we can test? Something like (not tested):
```
func shouldLog() (doLog, lastLog bool) {
  checkStats.M.RLock()
  defer checkStats.M.RUnlock()
  
  if s, found := checkStats.Stats[check.ID()]; found {
    lastlog = s.TotalRuns == 5
    doLog = s.TotalRuns <= 5 || s.TotalRuns%10 == 0
  } else {
    lastlog = false
    doLog = true
  }
}
```

Reply:
Yes I can do this

Reply:
I did this